client/src/components/new-request/send-to-examination.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <MainContainer id="non-analyze-container">
+    <template v-slot:container-header>
+      <v-col cols="auto" class="h6 pt-0 mt-1">
+        You are searching for a name for a
+        {{ entityText === ' BC Corporation' && location.text === ' BC' ? '' : ' ' + location.text }}
+        {{ entityText }}
+      </v-col>
+    </template>
+    <template v-slot:content>
+      <v-row class="mt-n2">
+        <NameInput :is-read-only="true"/>
+      </v-row>
+      <v-row class="text-center mt-n8">
+        <v-col cols="12" class="h5 text-center">
+          <v-icon :class="actionConfig.class">
+            {{ actionConfig.icon }}
+          </v-icon>
+          <span :class="actionConfig.class" >{{ actionConfig.text }}</span>
+        </v-col>
+        <v-col cols="12" class="mt-n4">
```

Comment:
Again, this is compensating for something that should be solved another way.

Reply:
Probably related, will have a closer look!

Reply:
fixed